<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>812</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/arrow-circle.png</normaloff>../../icons/bonus/icons-24/arrow-circle.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="UploadButton">
       <property name="text">
        <string>Upload</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/arrow-090.png</normaloff>../../icons/bonus/icons-24/arrow-090.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/arrow-270.png</normaloff>../../icons/bonus/icons-24/arrow-270.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="newdirButton">
       <property name="text">
        <string>New Dir</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/plus.png</normaloff>../../icons/bonus/icons-24/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="6" colspan="2">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/cross.png</normaloff>../../icons/bonus/icons-24/cross.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../icons/bonus/icons-24/arrow-circle-double.png</normaloff>../../icons/bonus/icons-24/arrow-circle-double.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="hostEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="3">
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLineEdit" name="portEdit"/>
     </item>
     <item row="1" column="11">
      <widget class="QCheckBox" name="sshBox">
       <property name="text">
        <string>SSH</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="12">
      <widget class="QWidget" name="infoWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
